packages/chaire-lib-backend/src/api/uploads.socketRoutes.ts

```diff
@@ -22,53 +39,137 @@ type ObjectImporter =
           type: 'importOnly';
       };
 
+// Remove anything before a slash or backslash in the filename to prevent path traversal attacks
+const sanitizeFileName = (filename: string) => filename.replace(/^.*[\\/]/, '');
```

Comment:
@tahini  Just to confirm, at this point, we would not need to handle %2F ? (Like in a param/name with %2e%2e%2f

Reply:
Right, we are not in an URL at this point, so this is ok